hugegraph-core/src/main/java/org/apache/hugegraph/util/ConfigUtil.java

```diff
@@ -112,9 +112,11 @@ public static Map<String, String> scanGraphsDir(String graphsDirPath) {
 
     public static String writeToFile(String dir, String graphName,
                                      HugeConfig config) {
-        E.checkArgument(FileUtils.getFile(dir).exists(),
+        File file = FileUtils.getFile(dir);
+        E.checkArgument(file.exists(),
                         "The directory '%s' must exist", dir);
-        String fileName = Paths.get(dir, graphName + CONF_SUFFIX).toString();
+        String tmpGraphName = graphName + CONF_SUFFIX;
```

Comment:
the security alert let us check the user params (trim it) before using: (reasonable)
<img width="749" alt="image" src="https://user-images.githubusercontent.com/17706099/202733423-5afeed0b-f28e-4638-b5b1-30cf154bc8c4.png">

Reply:
> in our case, fileName is "graphName.properties ", has one dot

we could assert only one `.` could exist, and another way is we just truncate before `.` then we add `.properties` as surffix later